client/src/pages/Login.js

```diff
@@ -146,23 +168,27 @@ const Login = ({ isLoginForm }) => {
         <WhiteSpace />
         <WhiteSpace />
         <SubmitButton
```

Comment:
There's a slight lighter blue in the border for the button (can see it when it's longer). I had this issue before, just need to override `border` from the default ANTD styling, either unset it or assign it to the correct color

<img width="199" alt="Screen Shot 2020-05-03 at 9 42 57 PM" src="https://user-images.githubusercontent.com/42228386/80936772-68687900-8d87-11ea-8f04-f0426a01ef11.png">


<img width="1124" alt="Screen Shot 2020-05-03 at 9 43 40 PM" src="https://user-images.githubusercontent.com/42228386/80936707-28a19180-8d87-11ea-8888-e88b46629afc.png">

Reply:
Neither unset nor setting it to the right color removes the line...

Reply:
I think we can leave it for now since the line only shows on larger screens

For another issue I'm thinking we could create reusable components like buttons and titles so that styles are easier to maintain. I had lots of troubling overriding ATND default styles so I think the reusable components could save everyone's time and sanity !!